rest/tests/test_rest.py

```diff
@@ -553,16 +558,23 @@ def test_immutable_fields(self):
             'date_created': "2015-01-01",
             'created_by': self.user_norepo.id,
         }
-        result = self.create_repository(repo_dict, skip_assert=True)
-        # these keys should be different since they are immutable or set by
-        # the serializer
-        self.assertNotEqual(repo_dict['id'], result['id'])
-        self.assertNotEqual(repo_dict['slug'], result['slug'])
-        self.assertNotEqual(repo_dict['date_created'], result['date_created'])
-        self.assertNotIn('created_by', result)
-        repository = Repository.objects.get(slug=result['slug'])
-        self.assertEqual(repository.created_by.id, self.user.id)
-        repo_dict = result
+
+        def assert_not_changed(new_dict):
+            """Check that fields have not changed"""
+            # these keys should be different since they are immutable or set by
```

Comment:
Nit: capitalization and punctuation missing. I'll stop commenting on writing stuff after this.